extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -137,6 +141,13 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
    * @private
    */
   buildInline_() {
+    if (this.doesContainFormElement_()) {
+      // Page attachments that contain forms must display the page's publisher
+      // domain above the attachment's contents. This enables users to gauge
+      // the trustworthiness of publishers before sending data to them.
+      this.headerEl.after(this.createDomainLabelElement_());
```

Comment:
Can you use `insertBefore` instead of `after`? I know it sounds weird but it has better browser support, check for examples in the codebase to see how we use it as `after`